controllers/vaultdynamicsecret_controller.go

```diff
@@ -183,15 +187,17 @@ func (r *VaultDynamicSecretReconciler) Reconcile(ctx context.Context, req ctrl.R
 
 	o.Status.SecretLease = *secretLease
 	o.Status.LastRenewalTime = time.Now().Unix()
+	o.Status.LastGeneration = o.GetGeneration()
 	if err := r.updateStatus(ctx, o); err != nil {
 		return ctrl.Result{}, err
 	}
 
 	reason := consts.ReasonSecretSynced
+	var horizon time.Duration
 	leaseDuration := time.Duration(secretLease.LeaseDuration) * time.Second
-	horizon := computeDynamicHorizonWithJitter(leaseDuration, o.Spec.RenewalPercent)
+	horizon = computeDynamicHorizonWithJitter(leaseDuration, o.Spec.RenewalPercent)
```

Comment:
super minor nit: I don't think this change is necessary?

Reply:
Think that was leftover from original patch for #187